tpu-client/src/nonblocking/tpu_client.rs

```diff
@@ -497,6 +520,28 @@ where
                         );
                         sleep(SEND_TRANSACTION_INTERVAL).await;
```

Comment:
this sleep is ineffective now? (and was probably dubious from the start)

Reply:
It's still used in between firing off all the messages, so it's not *totally* useless

Reply:
@lijunwangs this line here makes us sleep for 10ms in between sending transactions https://github.com/solana-labs/solana/pull/32844#discussion_r1297500248, which gives us a maximum of 100 transactions per second. 

Currently, if you deploy a program with a TPU client against devnet, sending one transaction takes between 100ms and 250ms.

Reply:
If I understand correctly, that sleep is for "re"-sending transactions which were sent before.

Reply:
This loop is going over all the transactions, and for each transaction, there's `send_wire_transaction_detached` and then `sleep`. The resending happens after all of the transactions have been sent